ios/FluentUI.Demo/FluentUI.Demo/Demos/OtherCellsDemoController.swift

```diff
@@ -32,6 +34,49 @@ class OtherCellsDemoController: DemoController {
     }
 }
 
+extension OtherCellsDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        var tokensClosure: ((ActionsCell) -> ActionsCellTokens)?
+        if isOverrideEnabled {
+            tokensClosure = { _ in
+                return ThemeWideOverrideActionsCellTokens()
+            }
+        }
+
+        fluentTheme.register(controlType: ActionsCell.self, tokens: tokensClosure)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        overrideTokens = isOverrideEnabled ? PerControlOverrideTableViewCellTokens() : nil
+        self.tableView.reloadData()
+    }
+
+    func isThemeWideOverrideApplied() -> Bool {
+        return self.view.window?.fluentTheme.tokenOverride(for: ActivityIndicatorCell.self) != nil
+    }
+
+    // MARK: - Custom tokens
+    private class ThemeWideOverrideActionsCellTokens: ActionsCellTokens {
+        override var regularTextColor: DynamicColor {
+            // "Charcoal"
+            return DynamicColor(light: GlobalTokens().sharedColors[.charcoal][.tint50],
+                                dark: GlobalTokens().sharedColors[.charcoal][.shade40])
+        }
+    }
+
+    private class PerControlOverrideTableViewCellTokens: TableViewCellTokens {
```

Comment:
Did you mean to have the theme wide and per control tokens be of different types?

Reply:
Yea, since I have different types of cells here I thought I'd demonstrate both this way

Reply:
It might be worth adding a second set of toggles for the second type of cell.